flow/Makefile

```diff
@@ -279,7 +279,19 @@ OPENROAD_GUI_CMD         = $(OPENROAD_EXE) -gui $(OR_ARGS)
 
 YOSYS_CMD               ?= $(abspath $(FLOW_HOME)/../tools/install/yosys/bin/yosys)
 
-KLAYOUT_CMD             ?= $(shell command -v klayout)
+# Use locally installed and built klayout if it exists, otherwise use klayout in path
+KLAYOUT_DIR = $(abspath $(FLOW_HOME)/../tools/install/klayout/)
+KLAYOUT_BIN_FROM_DIR = $(KLAYOUT_DIR)/klayout
+
+ifeq ($(wildcard $(KLAYOUT_BIN_FROM_DIR)), $(KLAYOUT_BIN_FROM_DIR))
+KLAYOUT_CMD ?= sh -c 'LD_LIBRARY_PATH=$(dir $(KLAYOUT_BIN_FROM_DIR)) $$0 "$$@"' $(KLAYOUT_BIN_FROM_DIR)
+else
+KLAYOUT_CMD ?= $(shell command -v klayout)
+endif
+
+KLAYOUT_CMD ?= $(if $(wildcard $(KLAYOUT_DIR)/klayout),\
+ LD_LIBRARY_PATH=$(KLAYOUT_DIR) $(KLAYOUT_DIR)/klayout,\
+ $(shell command -v klayout))
```

Comment:
It looks like KLAYOUT_CMD is defined and then redefined.  Please explain.

Reply:
the second is a harmless no-op accidentally left behind when tinkering(my eyes arent what they used to be,) and I should just delete that line.